<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="35530" started-at="2018-06-07T18:01:42Z" finished-at="2018-06-07T18:02:18Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="35530" started-at="2018-06-07T18:01:42Z" finished-at="2018-06-07T18:02:18Z">
      <class name="SeleniumTest.gmail.tests.GMailTest">
        <test-method status="PASS" signature="before()[pri:0, instance:SeleniumTest.gmail.tests.GMailTest@cb5822]" name="before" is-config="true" duration-ms="4497" started-at="2018-06-07T15:01:58Z" finished-at="2018-06-07T15:02:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="PASS" signature="gmailTest()[pri:0, instance:SeleniumTest.gmail.tests.GMailTest@cb5822]" name="gmailTest" duration-ms="7035" started-at="2018-06-07T15:02:03Z" finished-at="2018-06-07T15:02:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gmailTest -->
        <test-method status="PASS" signature="destroy()[pri:0, instance:SeleniumTest.gmail.tests.GMailTest@cb5822]" name="destroy" is-config="true" duration-ms="803" started-at="2018-06-07T15:02:10Z" finished-at="2018-06-07T15:02:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- destroy -->
      </class> <!-- SeleniumTest.gmail.tests.GMailTest -->
      <class name="SeleniumTest.gmail.tests.FailedLoginTest">
        <test-method status="PASS" signature="before()[pri:0, instance:SeleniumTest.gmail.tests.FailedLoginTest@18be83e4]" name="before" is-config="true" duration-ms="4252" started-at="2018-06-07T15:02:10Z" finished-at="2018-06-07T15:02:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="FAIL" signature="gmailTest()[pri:0, instance:SeleniumTest.gmail.tests.FailedLoginTest@18be83e4]" name="gmailTest" duration-ms="2405" started-at="2018-06-07T15:02:15Z" finished-at="2018-06-07T15:02:17Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login Failed expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login Failed expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertFalse(Assert.java:63)
	at SeleniumTest.gmail.tests.FailedLoginTest.gmailTest(FailedLoginTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gmailTest -->
        <test-method status="PASS" signature="destroy()[pri:0, instance:SeleniumTest.gmail.tests.FailedLoginTest@18be83e4]" name="destroy" is-config="true" duration-ms="804" started-at="2018-06-07T15:02:17Z" finished-at="2018-06-07T15:02:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- destroy -->
      </class> <!-- SeleniumTest.gmail.tests.FailedLoginTest -->
      <class name="SeleniumTest.ebay.tests.EBayTest">
        <test-method status="PASS" signature="before()[pri:0, instance:SeleniumTest.ebay.tests.EBayTest@1593948d]" name="before" is-config="true" duration-ms="5296" started-at="2018-06-07T15:01:42Z" finished-at="2018-06-07T15:01:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="PASS" signature="ebayTest()[pri:0, instance:SeleniumTest.ebay.tests.EBayTest@1593948d]" name="ebayTest" duration-ms="9604" started-at="2018-06-07T15:01:48Z" finished-at="2018-06-07T15:01:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ebayTest -->
        <test-method status="PASS" signature="destroy()[pri:0, instance:SeleniumTest.ebay.tests.EBayTest@1593948d]" name="destroy" is-config="true" duration-ms="816" started-at="2018-06-07T15:01:57Z" finished-at="2018-06-07T15:01:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- destroy -->
      </class> <!-- SeleniumTest.ebay.tests.EBayTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
